detox/src/utils/customConsoleLogger.js

```diff
@@ -11,7 +11,8 @@ function getStackDump() {
 
 function getOrigin() {
   const userCallsite = callsites()[USER_STACK_FRAME_INDEX];
-  const filename = path.relative(process.cwd(), userCallsite.getFileName());
+  const callsiteFilename = userCallsite && userCallsite.getFileName();
+  const filename = callsiteFilename ? path.relative(process.cwd(), callsiteFilename) : '<unknown>';
   return `at ${filename}:${userCallsite.getLineNumber() || '?'}:${userCallsite.getColumnNumber() || '?'}`;
```

Comment:
I am afraid that in case when userCallsite is indeed falsy, the attempt to get line and column number will fail as well. If you have started already guarding against this edge case, make sure to make it consistent here too.

Reply:
It didn't in my use case (I was surprised as well). I'll add logic tho.

Leaving test case as is cuz the line/col is obv non-deterministic (given my use case findings).  The update code will handle both scenarios.